source/incidents-and-alerts/index.html.md.erb

```diff
@@ -155,3 +155,31 @@ Throughout the incident, record actions taken, decisions, significant external c
 ## After the incident
 
 When the incident is resolved, the incident lead needs to record the completed template with a unique ID in the [Incident Log](https://docs.google.com/spreadsheets/d/1NKk_XgXFooHSLlnKtIGMsWeKy9Ws3WU61VtcRdH2OIE/edit).
+
+## Helping other organisation incident management teams with their incidents
+
+From time to time GovWifi may be requested to assist an organisation that is in incident management mode. The following principles should help make the engagement easier.
+
+### Identify the admin user for the organisation
+
+Within the admin portal, locate the admin user listed for the organisation in question. The incident lead for the organisation is unlikely to be our admin user so confirm that the admin user is also available in any resolution call.
+
+### Understand the structure of the other organisation incident team
```

Comment:
This section feels like it's doing a few too many things at once, and I don't think this heading completely reflects the content below.

Would suggest changing this section to something like: 

"### Introduce everyone involved in the incident

When speaking to the organisation's incident team, make sure everyone has been properly introduced. The team may have a very different structure to ours, and we need to understand their roles and hierarchy. "